docs/reference-guide/cookieplone-make-commands.md

```diff
@@ -9,88 +9,146 @@ myst:
 
 # Cookieplone `make` commands
 
-This guide explains all the `make` commands found in a cookieplone project.
+This reference guide describes the function and purpose of all the `make` commands found in a {term}`Cookieplone` project.
+It's organized according to its components.
+-   Frontend
+-   Backend
+-   Documentation
+-   Environment
+-   Quality assurance (QA)
+-   Internationalization (i18n)
+-   Testing
+-   Container images
+-   Local stack
+-   Acceptance
 
 ## Frontend
 
-The Makefile at the root of your project invokes commands in frontend/Makefile. Here are excerpts from frontend/Makefile to show details of the frontend make commands.
+When you issue a `make` command at the root of your project, you call the file {file}`Makefile` also at the root.
+In turn, it invokes commands in the file {file}`frontend/Makefile`.
+You can refer to these files for implementation details.
+You can run the following make targets by using the command structure of `make <TARGET>`.
 
 `frontend-install`
+:   Invokes the target `make install` in `frontend/Makefile`.
```

Comment:
For this and all targets, please remove `make `. I must have been half asleep when I suggested that. `make install` is a command. `install` is a make target. See line 30 and https://makefiletutorial.com/

```suggestion
:   Invokes the target `install` in `frontend/Makefile`.
```